src/code/entities/bank3.asm

```diff
@@ -2367,115 +2426,132 @@ SpawnEnemyDrop::
     ldh  a, [hActiveEntityType]                   ; $55CF: $F0 $EB
```

Comment:
Is b of the bc input allways 0?
What are the possible values for bc?

Reply:
As the maximum number of entities is 16, the entity index can range from `$00` to `$0F`. So yes, in that case the `b` register will always be zero.

(But the argument used by the function code to look up the entity in the table is `bc`, so `b` it is still technically part of the input.)

Reply:
ok this simplify the code more

Reply:
where can I find what the values are for?

Reply:
For the entity index? Well, it's only an index. Entities are referred to by index in the table of entities, so the code will often say "Let's update entity in the slot n° 3", and use 3 as the reference of the entity being updated.

Or do you mean something else?

Reply:
sure the function of the index i understand.
c is the index value here, but what stands the value $00, $01, $02, ... for?
I mean the index of the entity as used in entities.asm it could not be. there are more than 16 and the first 16 would make no sence here